public/themes/custom/hdbt_subtheme/templates/views/views-view--daycare-search.html.twig

```diff
@@ -49,10 +49,9 @@
   {{ attachment_before }}
 
   <div class="unit-search__results" data-id-number="{{ dom_id }}">
-    <span class="unit-search__count-container">
-      <span class="unit-search__count">{{ total_rows }}</span>
-      {% trans with {'context': 'Daycare search count'}%}location{% plural total_rows %}locations{% endtrans %}
-    </span>
+    <h3 class="unit-search__count-container">
+      {{ total_rows }} {% trans with {'context': 'Daycare search count'}%}daycare centre{% plural total_rows %}daycare centres{% endtrans %}
```

Comment:
`total_rows` can be null if there are no search results. Passing null to `plurar` causes log spam.

I was just looking into this yesterday when I was browsing the logs 😅:
https://github.com/City-of-Helsinki/drupal-helfi-rekry/pull/388

Reply:
Resolving this conversations since I think the log spam should be fixed here: https://github.com/City-of-Helsinki/drupal-hdbt/blob/main/hdbt.theme#L1290-L1292.